code/modules/mob/living/carbon/human/species.dm

```diff
@@ -150,8 +150,11 @@
 
 
 /datum/species/proc/hug(var/mob/living/carbon/human/H,var/mob/living/target)
-
-	H.visible_message("<span class='notice'>[H] hugs [target] to make [target.p_them()] feel better!</span>", \
+	if(H.zone_selected == "head")
+		H.visible_message("<span class='notice'>[H] pats [target] on the head affectionately.</span>", \
+					"<span class='notice'>You pat [target] on the head affectionately.</span>", null, 4)	
```

Comment:
I'd say remove the "affectionately" part, as a pat in the head can have multiple meanings. Could be an affectionate one, but also a patronizing one and whatnot.

Reply:
I like the way you think.

Reply:
Indeed. Made the change.